store/backup.go

```diff
@@ -495,6 +499,7 @@ func (s *Store) findBackupImpl(ctx context.Context, tx *sql.Tx, find *api.Backup
 			&backupRaw.MigrationHistoryVersion,
 			&backupRaw.Path,
 			&backupRaw.Comment,
+			&backupRaw.Payload,
```

Comment:
Have we updated the database schema?

Reply:
Yes, it's already here
https://github.com/bytebase/bytebase/blob/b43d60027296c184c0ebd80f9b643bbd475fa667/store/migration/dev/LATEST.sql#L545